doc/user_guide/tools.rst

```diff
@@ -240,13 +240,48 @@ The methods of this section are available to all the signals. In other chapters
 methods that are only available in specialized
 subclasses.
 
+Simple mathematical operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionchaned:: 0.9
```

Comment:
No, I was meaning adding that little description inside  `versionchanged` not to confuse the reader in thinking that all that is below is new in 0.9.